bin/cli.js

```diff
@@ -132,6 +132,14 @@ function dependencyFilter() {
 	};
 }
 
+function getPackageConfig() {
+	try {
+		return require(path.join(process.cwd(), 'package.json')).madge;
```

Comment:
It think this is fine but move the require to the top where the other requires are.

Reply:
fixed. I think it easier to read.

Reply:
Sweet :)